test/local-e2e-tests.sh

```diff
@@ -22,70 +22,73 @@ dir=$(dirname "${BASH_SOURCE[0]}")
 base=$(cd "$dir/.." && pwd)
 
 # Strimzi installation config template used for starting up Kafka clusters.
-readonly STRIMZI_INSTALLATION_CONFIG_TEMPLATE="test/config/100-strimzi-cluster-operator-0.20.0.yaml"
+readonly LOCAL_STRIMZI_INSTALLATION_CONFIG_TEMPLATE="test/config/100-strimzi-cluster-operator-0.20.0.yaml"
```

Comment:
I wonder if we could source `common.sh` and reuse those functions for local run as well, not necessarily right now it can be postponed to future iterations.